### PR TITLE
Add French Images, fix locations user assignment issue

### PR DESCRIPTION
**Pull request by bennsimon**

### CODE REVIEW

File: opensrp-eusm/src/main/java/org/smartregister/eusm/repository/AppStructureRepository.java [hunk 1]
```diff
@@ -156,6 +157,11 @@ public int countOfStructures(String nameFilter, String locationParentId) {
 
 
         Location location = EusmApplication.getInstance().getUserLocation();
+        if (location == null) {
+            location = new Location("temp");
+            location.setLongitude(0.000);
+            location.setLatitude(0.000);
+        }
```
Comment (githengi) [inline opensrp-eusm/src/main/java/org/smartregister/eusm/repository/AppStructureRepository.java:164]: why is this needed>

Comment (githengi) [reply opensrp-eusm/src/main/java/org/smartregister/eusm/repository/AppStructureRepository.java:164]: Seen you are calculating distances on sqlite. Its not possible to calculate accurate distances on sqlite since it lacks some cos/sin/tan functions

Comment (bennsimon) [reply opensrp-eusm/src/main/java/org/smartregister/eusm/repository/AppStructureRepository.java:164]: I use that as the default location, ideally the app should work without it.

**Review by githengi — APPROVED** [review]

File: opensrp-eusm/src/main/java/org/smartregister/eusm/repository/AppStructureRepository.java (hunk 1 above)
Comment (bennsimon) [reply opensrp-eusm/src/main/java/org/smartregister/eusm/repository/AppStructureRepository.java:164]: I use it a default location but when the app is being used it will be replaced by the actual location.
